Iazel/RegenProductUrl/Service/RegenerateProductUrl.php

```diff
@@ -70,23 +74,25 @@ public function __construct(
     }
 
     /**
-     * @param int[] $productIds
-     * @param int   $storeId
+     * @param int[]|nul $productIds
+     * @param int|nul $storeId
+     * @param bool $verbose
      *
      * @return void
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
      */
-    public function execute(array $productIds, int $storeId): void
+    public function execute(?array $productIds = null, ?int $storeId = null, bool $verbose = false): void
     {
         $this->regeneratedCount = 0;
-        $stores                 = $this->storeManager->getStores(false);
+
+        $stores = isNull($storeId)
```

Comment:
`isNull` checks whether a FFI\CData is a null pointer. You'll want to use `is_null` here.

Reply:
sorry... I fixed it.